tests/Base.php

```diff
@@ -161,6 +161,24 @@ public function testRuntimeGarbageBody(): void
 
     public function testConsoleLogs(): void {
         $response = $this->call([]);
-        self::assertStringStartsWith("log1\n{hello: world}\n[hello, world]", $response['body']['stdout']);
+        
+        // TODO: @Meldiron Test stderr
```

Comment:
`stderr` is only supported in some languages, like Node with `console.error`. Tho not supported in all languages. I would like to avoid stderr tests until we have proper logging interface.

Reply:
I will make sure to implement propers tests in v3 PR: https://github.com/open-runtimes/open-runtimes/pull/130

Reply:
okay